backends/cuda/ceed-cuda-compile.h

```diff
@@ -13,6 +13,7 @@
 static inline CeedInt CeedDivUpInt(CeedInt numerator, CeedInt denominator) { return (numerator + denominator - 1) / denominator; }
 
 CEED_INTERN int CeedCompile_Cuda(Ceed ceed, const char *source, CUmodule *module, const CeedInt num_defines, ...);
+CEED_INTERN int CeedTryCompile_Cuda(Ceed ceed, const char *source, bool *is_compile_good, CUmodule *module, const CeedInt num_defines, ...);
```

Comment:
Here I'm adding this (and below) so we can do a try-catch sort of thing in the operator code generation. Since we have an immediate fallback on failure, I'm not worried about losing an actually important error.